tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/UpdateCombinationStockFeatureContext.php

```diff
@@ -145,6 +145,19 @@ public function assertLastErrorIsDuplicateQuantityUpdate(): void
         );
     }
 
+    /**
+     * @Then I should get error that stock available quantity is invalid
+     *
+     * @return void
+     */
+    public function assertLastErrorIsInvalidStockAvailableQuantity(): void
```

Comment:
Perfect for the combination use case, but we should do the same on a standard product as well

Reply:
Suggestion, not mandatory, but since this endpoint is used for both standard products and products with combination maybe it should be moved into `UpdateStockFeatureContext` which is a bit more generic and it already contains a few endpoints related to `ProductStockConstraintException`